src/main/res/values/strings.xml

```diff
@@ -197,10 +197,12 @@
     <string name="language_fr">French</string>
     <string name="language_es">Spanish</string>
     <string name="language_bn">Bengali</string>
+    <string name="language_hi_rIN">Hindi</string>
```

Comment:
remove _rIN from the key since we are following "hi" pattern and not "hi_rIN". So it's better to be consistent with variable names. See how it's done for Bengali for reference.